shuffle/cli/tests/shuffle-forge-cli-tests.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+mod common;
+
+use forge::{
+    forge_main, AdminContext, AdminTest, ForgeConfig, LocalFactory, Options, Result, Test,
+};
+
+use common::{bootstrap_shuffle, ShuffleTestHelper};
+use smoke_test::scripts_and_modules::enable_open_publishing;
+use std::{os::unix::prelude::ExitStatusExt, process::ExitStatus};
+
+pub const BINARY: &str = env!("CARGO_BIN_EXE_shuffle");
+
+fn main() -> Result<()> {
+    let tests = ForgeConfig::default().with_admin_tests(&[
+        &TransactionsWithoutNetworksToml,
+        &TransactionsWithoutAccount,
+        &TransactionsWithNetworkRawAddressFlags,
+    ]);
+    let options = Options::from_args();
+    forge_main(tests, LocalFactory::from_workspace()?, &options)
+}
+
+pub struct TransactionsWithoutNetworksToml;
+
+impl Test for TransactionsWithoutNetworksToml {
+    fn name(&self) -> &'static str {
+        "shuffle::transactions-without-networks-toml"
+    }
+}
+
+impl AdminTest for TransactionsWithoutNetworksToml {
+    fn run<'t>(&self, ctx: &mut AdminContext<'t>) -> Result<()> {
+        let helper = ShuffleTestHelper::new(ctx.chain_info())?;
+        let home_path_string = helper.home_path().to_string_lossy().to_string();
+        let output = std::process::Command::new(BINARY)
+            .args([
+                "--home-path",
+                home_path_string.as_str(),
+                "transactions",
+                "--network",
+                "forge",
+            ])
+            .output()?;
+        let std_err = String::from_utf8(output.stderr)?;
+        assert!(std_err
+            .contains("Error: Please add specified network to the ~/.shuffle/Networks.json\n"));
+        assert_eq!(output.status, ExitStatus::from_raw(256));
+        Ok(())
+    }
+}
+
+pub struct TransactionsWithoutAccount;
+
+impl Test for TransactionsWithoutAccount {
+    fn name(&self) -> &'static str {
+        "shuffle::transactions-without-account"
+    }
+}
+
+impl AdminTest for TransactionsWithoutAccount {
+    fn run<'t>(&self, ctx: &mut AdminContext<'t>) -> Result<()> {
+        let helper = ShuffleTestHelper::new(ctx.chain_info())?;
+        let home_path_string = helper.home_path().to_string_lossy().to_string();
+        let client = ctx.client();
+        let factory = ctx.chain_info().transaction_factory();
+        enable_open_publishing(&client, &factory, ctx.chain_info().root_account())?;
+        helper.create_project()?;
```

Comment:
why do you need a project to run `shuffle transactions`? Is this you trying to force the `overwrite_networks_config...`?

Reply:
yes. removing now that overwrite_networks_config is in the ShuffleTestHelper::new()